scheds/rust/scx_lavd/src/bpf/intf.h

```diff
@@ -21,8 +21,11 @@ typedef signed long s64;
 
 typedef int pid_t;
 
+#define U64_MAX ((u64)~0ULL)
+
 enum {
 	TASK_COMM_LEN = 16,
+	SCX_SLICE_INF = U64_MAX,
```

Comment:
Hmm... this should be available in `vmlinux.h` but I guess there's no easy way to expose that to the userland part. I'll think about how this can be solved in a better way. Not an issue with your PR in any way.

Reply:
Exactly, there is no easy way to expose some kernel definitions to the userland. It might be better a header into two files -- one for common between userland and kernel and another for kernel only.

Reply:
Yeah, this has been bothering me for a while. I'll try to see if there's a nice solution that can be shared by everyone.